deoxys/Cargo.toml

```diff
@@ -35,6 +35,7 @@ bytes = ["aead/bytes"]
 os_rng  = ["aead/os_rng", "rand_core"]
 heapless = ["aead/heapless"]
 rand_core = ["aead/rand_core"]
+inout = ["aead/inout"]
```

Comment:
Why are you making this a feature? It's already a hard dependency of `aead`

Reply:
I started this way before I realized it didn't make sense, that shouldn't have gone through, sorry.